dts/sdm450-htc-brepdugl.dts

```diff
@@ -0,0 +1,23 @@
+ 
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8952.dtsi"
```

Comment:
Is this correct?

Reply:
use msm8952.dtsi, but does it have different with msm8953.dtsi? thanks!

Reply:
They are the same, but you should use msm8953.dtsi anyways, 8952 is another SOC.